assets/js/main.js

```diff
@@ -102,11 +118,8 @@ function luoUusiDemooni(i) {
 function tapaDemooni(demooni, pakota = false) {
     if (teemat[teema].kuolevainen || pakota) {
         demooni.innerHTML = "&#x1F4A5;";
+        demooni.classList.add("kuaallu");

```

Comment:
This is some weird dialect and doesn't follow our naming scheme that much but I think it's fine. :smiley: